crates/edr_provider/src/requests/serde.rs

```diff
@@ -351,7 +351,7 @@ where
 ///
 /// While technically the Yellow Paper allows specifying the timestamp as
 /// any scalar value, using `u64` in practice is more than enough.
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
```

Comment:
drive-by - since it's 8 bytes on 64-bit machine (our only targets?) it's cheaper to copy it around.